app/api/dao/mentorship_relation.py

```diff
@@ -92,23 +93,20 @@ def create_mentorship_relation(self, user_id, data):
         return {'message': 'Mentorship relation was sent successfully.'}, 200
 
     @staticmethod
+    @email_verification_required
     def list_mentorship_relations(user_id=None, accepted=None, pending=None, completed=None, cancelled=None, rejected=None):
-        if pending is not None:
```

Comment:
Why?

Reply:
@ramitsawhney27 both works similar but ```==``` is better than ```is``` 
**Source:** [Difference between 'is' and '==' operator](https://www.geeksforgeeks.org/difference-operator-python/)

Reply:
One operator isn't better than the other. Please read the link you've specified carefully and, make the necessary changes. Please don't blindly make changes.